Mathlib/SetTheory/Ordinal/FixedPointApproximants.lean

```diff
@@ -67,7 +67,9 @@ variable [CompleteLattice α] (f : α →o α) (x : α)
 open Function fixedPoints Cardinal Order OrderHom
 
 set_option linter.unusedVariables false in
-/-- Ordinal approximants of the least fixed point greater then an initial value x -/
+/-- The ordinal-indexed sequence approximating the least fixed point greater than
+  an initial value `x`. It is defined in such a way that we have `lfpApprox 0 x = x` and
+  `lfpApprox a x = ⨆ b < a, f (lfpApprox b x)`. -/
```

Comment:
```suggestion
an initial value `x`. It is defined in such a way that we have `lfpApprox 0 x = x` and
`lfpApprox a x = ⨆ b < a, f (lfpApprox b x)`. -/
```
Please don't indent docstrings

Reply:
Whoops, this PR predates when you first told me not to do that :P